src/main/java/frc/robot/subsystems/flywheel/FlyWheelIOVariable.java

```diff
@@ -0,0 +1,90 @@
+package frc.robot.subsystems.flywheel;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.DemandType;
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.DoubleSolenoid.Value;
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+
+import edu.wpi.first.math.util.Units;
+import frc.robot.Constants;
+
+public class FlyWheelIOVariable implements FlyWheelIO {
+
+    private static final double encoderTicksPerRev = 2048.0;
+
+    private WPI_TalonFX leftFlyWheel;
+    private WPI_TalonFX rightFlyWheel;
+    private WPI_TalonFX topFlyWheel;
+    private DoubleSolenoid flyWheelExtender;
+    
+    public static boolean isExtended;
```

Comment:
Why is this static?

Reply:
because I believe we make a static reference to it. Would it be an issue if it was? It may cause memory issues but I don't think we will be using it that many times in a match to do that

Reply:
static reference in Telefly